admin_guide/persistent_storage/persistent_storage_aws.adoc

```diff
@@ -0,0 +1,128 @@
+= Persistent Storage Using AWS ElasticBlockStore
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+OpenShift supports AWS Elastic Block Store volumes (EBS). You can provision your OpenShift cluster with
+link:../../architecture/additional_concepts/storage.html[persistent storage] using
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html[AWS EC2].
+Some familiarity with Kubernetes and AWS is assumed.
+
+[IMPORTANT]
+====
+Before creating persistent volumes using AWS, OpenShift must first be
+properly link:../configuring_aws.html[configured for AWS ElasticBlockStore].
+====
+
+The Kubernetes
+link:../../architecture/additional_concepts/storage.html[persistent volume]
+framework allows administrators to provision a cluster with persistent storage
+and gives users a way to request those resources without having any knowledge of
+the underlying infrastructure. Persistent volumes are not bound to a single
+project or namespace; they can be shared across the OpenShift cluster.
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
+volume claims], however, are specific to a project or namespace and can be
+requested by users.
+
+
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
+====
+
+[[aws-provisioning]]
+
+== Provisioning
+Storage must exist in the underlying infrastructure before it can be mounted as
+a volume in OpenShift. After ensuring OpenShift is
+link:../../configuring_aws.html[configured for AWS ElasticBlockStore], all that is
```

Comment:
Should be `link:../../admin_guide/configuring_aws.html[configured for AWS ElasticBlockStore]`